listenbrainz/webserver/templates/user/import.html

```diff
@@ -30,6 +30,13 @@ <h2 class="page-title">Import to user {{ user.musicbrainz_id }}</h2>
         </form>
       </div>
 
+    <p>If you want to reset your previous import timestamp, click below</p>
```

Comment:
I think we need more verbiage here. "If you think that a partial import has somehow missed some listens, you may reset your previous import timestamp. This will cause your next import to be a complete import which should add any missing listens while still avoiding adding duplicates to your history."